alpaca/trading/client.py

```diff
@@ -661,3 +664,53 @@ def get_corporate_announcement_by_id(
             return response
 
         return CorporateActionAnnouncement(**response)
+
+    # ############################## OPTIONS CONTRACTS ################################# #
+
+    def get_option_contracts(
+        self, request: GetOptionContractsRequest
+    ) -> Union[OptionContractsResponse, RawData]:
+        """
+        The option contracts API serves as the master list of option contracts available for trade and data consumption from Alpaca.
+
+        Args:
+            request (GetOptionContractsRequest): The parameters that option contracts can be queried by.
+
+        Returns:
+            OptionContracts (Union[OptionContractsResponse, RawData]): The object includes list of option contracts.
+        """
+        if request is None:
+            raise ValueError("request (GetOptionContractsRequest) is required")
+        if request.underlying_symbol == "":
+            raise ValueError("underlying_symbol is required")
+
+        params = request.to_request_fields()
+
+        response = self.get(f"/options/contracts", params)
```

Comment:
f-string that does not have any interpolated variable

Reply:
Thank you! Fixed!